src/main/java/org/broadinstitute/hellbender/tools/exome/ACNVCopyRatioModeller.java

```diff
@@ -362,13 +362,13 @@ private CopyRatioDataCollection(final SegmentedModel segmentedModel) {
                 throw new IllegalArgumentException("Cannot construct CopyRatioDataCollection with no target-coverage data.");
             }
             //construct coverages and number of targets per segment from the segments and Genome in SegmentedModel
-            final TargetCollection<TargetCoverage> targetCoverages = segmentedModel.getGenome().getTargets();
+            final TargetCollection<ReadCountRecord.SingletonRecord> targetCoverages = segmentedModel.getGenome().getTargets();
```

Comment:
`Genome` stores its targets in a`TargetCollection`, so this is what you get when query the `SegmentedModel` for its targets.  Although this particular code would probably be fine if `Genome` stored `Targets` in a `ReadCountCollection`, there are other places where the ability of a `TargetCollection` to give all targets in a given interval is needed.

It might make sense to add this capability to `ReadCountCollection`, but I think that's beyond the scope of this ticket.